### PR TITLE
Greece: Updated region records

### LINKED ISSUE #1145: Greece: region updates
Greece should have 14 regions (13 regions + 1 autonomous state). WOF now has 12 records.

https://spelunker.whosonfirst.org/id/85633171/descendants/?exclude=nullisland&placetype=region

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1145.

This PR includes the following updates:

- A new `region` record for "North Aegean" in Greece
- Updated hierarchies/parent ids for any new descendant of North Aegean 
- Updated English `wof:name` values for all other regions, with existing names stored as `name:ell_x_*` property values
- Adds a region record for the autonomous region of [Mt. Athos](https://en.wikipedia.org/wiki/Mount_Athos) ([1](https://en.wikipedia.org/wiki/Administrative_regions_of_Greece), [2](http://www.statoids.com/ugr.html))
- Cuts up the existing geometry of [East Macedonia and Thrace](https://spelunker.whosonfirst.org/id/85684701/) to assign the various islands to the correct region

Regions of Greece:

```
data/856/846/59/85684659.geojson
data/856/847/01/85684701.geojson
data/856/846/97/85684697.geojson
data/856/846/27/85684627.geojson
data/856/846/21/85684621.geojson
data/856/846/81/85684681.geojson
data/856/846/31/85684631.geojson
data/856/846/85/85684685.geojson
data/856/846/93/85684693.geojson
data/856/846/37/85684637.geojson
data/856/846/55/85684655.geojson
data/856/846/43/85684643.geojson
data/136/066/603/9/1360666039.geojson
data/136/066/603/1/1360666031.geojson
```

### CODE REVIEW

File: data/136/066/603/1/1360666031.geojson [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "geom:longitude":26.105232,
+    "iso:country":"GR",
+    "lbl:latitude":39.188685,
+    "lbl:longitude":25.986438,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:max_zoom":11.0,
+    "mz:min_zoom":7.7,
+    "name:eng_x_preferred":[
+        "North Aegean"
```
Comment (nvkelso) [inline data/136/066/603/1/1360666031.geojson]: Please also add a Greek preferred name for new North Aegean.

**Comment by nvkelso** [comment]
> Add a region record for the autonomous region of Mt. Athos (1, 2)
> Cut up the existing geometry of East Macedonia and Thrace to assign the various islands to the correct region

Yes please for both.

File: data/136/066/603/1/1360666031.geojson (hunk 1 above)
Comment (stepps00) [reply data/136/066/603/1/1360666031.geojson]: Added in f085a13

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/1335#issuecomment-426790084 taken care of in 5c1da5a and bb3f9ce. 

This PR should be ready for review once I commit the PIP work to all Greece descendants.

**Comment by stepps00** [comment]
b14a914 updates hierarchies and parent ids for descendants of Greece regions.

Ready for a second glance.

**Comment by stepps00** [comment]
New region geometries, labelled with their new `wof:name`:

screen shot 2018-10-09 at 9 36 22 am

This PR should now be mergeable. @nvkelso

**Comment by nvkelso** [comment]
@stepps00 Where's the new `North Aegean` region?

**Comment by stepps00** [comment]
@nvkelso added North Aegean to the screenshot

**Pull request merged**